modules/apps/feature-flag/feature-flag-web/src/main/resources/META-INF/resources/js/FeatureFlagToggle.tsx

```diff
@@ -55,6 +55,14 @@ const FeatureFlagToggle = ({
 						? data.dependentFeatureFlags
 						: []),
 				]);
+
+				const toggleButton = document.querySelector(
```

Comment:
We should be able to use `useRef` instead of querying the dom for this.

Reply:
Fixed